src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");
```

Comment:
The message in the 'doWork' method should be more informative. It should indicate that it's an 'Excavator' that started working, in line with the checklist requirement for informative messages.